azerothcore/azerothcore-wotlk
# fix(Scripts/HoL): fix IsEncounterInProgress()

## Changes Proposed:
-  fix `IsEncounterInProgress()` in `instance_halls_of_lightning.cpp`, which returned `true` after Loken starts his speech.

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/6066

## SOURCE:


## Tests Performed:

- Applied C++ fix and tested on Windows 10 clients and server

## How to Test the Changes:


1. Get two characters in a group
2. With both characters, `.go 1126 34 65 602`
3. Wait for Loken to do his intro
4. Wipe
5. With one character, release spirit, run back to the instance and return to Loken (`.go 1126 34 65 602`)
6. Learn a resurrection spell (`.learn 2008` for example)
7. Verify that you can revive the other character

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
